pages/servers/search.tsx

```diff
@@ -52,9 +52,13 @@ const Search:NextPage<SearchProps> = ({ serverData, query }) => {
 export const getServerSideProps = async(ctx: Context) => {
 	if(ctx.query.query && !ctx.query.q) ctx.query.q = ctx.query.query
 	if(!ctx.query?.q) {
-		ctx.res.statusCode = 301
-		ctx.res.setHeader('Location', '/')
-		return { props: {} }
+		return {
```

Comment:
https://nextjs.org/docs/api-reference/next.config.js/redirects#header-cookie-and-query-matching